examples/evm/seth.py

```diff
@@ -334,13 +402,74 @@ def compare_buffers(a, b):
                     return False
             return cond
 
+        if state_id == -1:
+            state = self.initial_state
+        else:
+            state = self._executor._workspace.load_state(state_id, delete=False)
+
+        world = state.platform
+        trace = state.context['seth.trace']
+        last_pc = trace[-1][1]
+        last_address = trace[-1][0]
+        try:
+            md_name, md_source_code, md_init_bytecode, md_metadata, md_metadata_runtime, md_hashes= self.context['seth']['metadata'][last_address]
+        except:
+            md_name, md_source_code, md_init_bytecode, md_metadata, md_metadata_runtime, md_hashes = None,None,None,None,None,None 
+
+
+        try:
+            runtime_bytecode = world.storage[last_address]['code']
+        except:
+            runtime_bytecode = ''   
+
+        e = state.context['last_exception']
         if ty is not None:
             if str(e) != ty:
                 return
         print "="*20
-        print "REPORT:", e, "\n"
+        print "REPORT:", e, 
+
+        try:
+            asm = list(evm.EVMDecoder.decode_all(runtime_bytecode[:-9-33-2]))
```

Comment:
what is this magic number?

Reply:
http://solidity.readthedocs.io/en/develop/metadata.html#encoding-of-the-metadata-hash-in-the-bytecode

Reply:
I wanted to show that we can handle generic transactions. That's an evm transaction.
With 4 bytes of symbolic data (input)
And symbolic value